dist/actions/BoardActions.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.loadBoard = undefined;
+
+var _reduxActions = require('redux-actions');
+
+var loadBoard = exports.loadBoard = (0, _reduxActions.createAction)('LOAD_BOARD');
```

Comment:
There is a separate bower build workflow that uses browserify to build a single index.js bundle inside dist folder. Please remove all new files from dist folder as part of this PR